### DISCUSSION TITLE
why my Controls is not show description and defaultValue？

### DISCUSSION THREAD

**Discussion by 314171441** [storybookjs/storybook#18631] [Help]
**why my Controls is not show description and defaultValue？
But just show in Docs.**

**my: **
```
argTypes: {
    fullWidth: {
      control: {
        type: 'boolean',
      },
      description: 'width is or not 100%',
      table: {
        type: {
          summary: 'boolean | undefined',
        },
        defaultValue: {
          summary: false,
        },
        category: 'style',
      },
    },
    error: {
      control: {
        type: 'boolean',
      },
      description: 'is error',
      table: {
        type: {
          summary: 'boolean | undefined',
        },
        defaultValue: {
          summary: false,
        },
        category: 'style',
      },
    },
  },
```
![屏幕截图 2022-07-05 233200](https://user-images.githubusercontent.com/39342331/177364878-d42c4431-d43d-401c-a4d5-9ebab2748f90.png)
![屏幕截图 2022-07-05 233303](https://user-images.githubusercontent.com/39342331/177364899-9c3e1ab6-a6a1-4f78-b7b8-9aa12c8c99a5.png)

**my expectation: **

![1211](https://user-images.githubusercontent.com/39342331/177365118-da9d7df6-6df9-476b-9652-c67727713eaa.png)

**Comment by etedesco** [comment]
With SB 8.3 I have the same issue.
